server/src/Logger.ts

```diff
@@ -0,0 +1,29 @@
+import * as Express from "express";
+import * as morgan from "morgan";
+import * as fs from "fs";
+import * as path from "path";
+
+/**
+ * The loggerApi handles each request and logs it to a file called requests.log in the root of the server.
+ * If the response status code is > 400, the request is logged to the errors.log file at the root of 
+ * the server.
+ * @param {format: string} the format that Morgan uses to output request logs
+ */
+export const loggerApi = ( format: string ): Express.RequestHandler[] => {
+    const requestStream = fs.createWriteStream( path.join( __dirname, '..', 'requests.log' ), { flags: 'a' } );
+    const errorStream = fs.createWriteStream( path.join( __dirname, '..', 'errors.log' ), { flags: 'a' } );
+
+    return [
+        morgan( format, {
+            skip: ( req: Express.Request, res: Express.Response ) => {
+                return res.statusCode >= 400;
```

Comment:
500 server errors will go into your requestStream instead of the errorStream

Reply:
It seems kind of weird but the way that morgan works is the `skip` thing is a function which, if it returns true, will skip the stream and move on to the next middleware function. If it returns false, as is the case with a response stats code of < 400, it will use the requestStream. The next middleware function is there the errors are caught because they fail the test. It's slightly backwards logic i know,  but it's how morgan works.